File: westcoast/src/main/java/frc/motorFactory/StopAllMotors.java
package frc.motorFactory;
import edu.wpi.first.wpilibj.SpeedController;
import edu.wpi.first.wpilibj.SpeedControllerGroup;
public class StopAllMotors {
    private SpeedControllerGroup motors;
    public StopAllMotors(
        SpeedController motor1,
        SpeedController motor2,
        SpeedController motor3
    ){
        motors = new SpeedControllerGroup(motor1, motor2, motor3);
    }
    public StopAllMotors(
        SpeedController motor1,
        SpeedController motor2
    ){
        this(motor1, motor2, motor1);
    }
    public void stop(){
        motors.stopMotor();
    }
}

File: westcoast/src/main/kotlin/frc/commands/motors/StopAllMotors.kt
package frc.commands.motors

import edu.wpi.first.wpilibj.SpeedController
import edu.wpi.first.wpilibj2.command.InstantCommand

class StopAllMotors constructor(
    private val motor1: SpeedController,
    private val motor2: SpeedController,
    private val motor3: SpeedController,
) : InstantCommand() {
  override fun execute() {
    motor1.stopMotor()
    motor2.stopMotor()
    motor3.stopMotor()
  }
}